data/blog/rene-pickhardt-receives-lts-grant.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: "Long-Term Support For René Pickhardt"
+date: '2024-01-18'
+tags: ['OpenSats', 'grants', 'bitcoin', 'LTS']
+draft: false
+authors: ['cfunk']
+images: ['/static/images/blog/22-rene.jpg']
+summary: "OpenSats Welcomes René Pickhardt as an LTS Grantee."
+---
+
+
+We are very excited to award a Long-term Support Grant to René Pickhardt,
+helping him continue his fundamental research on improving Bitcoin’s overall
+usefulness as a payment system through the Lightning Network.
+
+Fueled by gratitude for his scholarship from the [German National Academic
+Foundation](https://en.wikipedia.org/wiki/Studienstiftung), and the everyday
+benefits of open-source software he uses, René has centered his career on giving
+back to the communities that supported him on his path.
+
+He has consistently shared commercially significant knowledge for a long time
+without seeking prior funding. All major LN implementations have integrated some
+of his research results and algorithms, now widely recognized as the most
+optimal for bitcoin payments.
+
+Over the years, he has achieved an almost endless list of invaluable
+contributions, including creating [Pickhardt
+Payments](https://github.com/renepickhardt/pickhardtpayments), renepay, actively
```

Comment:
I think I only helped answering questions to the developer who created renepay

Reply:
maybe best to just scratch that one off the list for the sake of simplicity